rbac/internal/utils.py

```diff
@@ -30,6 +33,14 @@
 
 logger = logging.getLogger(__name__)
 
+HOST = os.getenv("HOST")
```

Comment:
it is needed to put var in setting and clower yaml, so we can use it in ephemeral, stage and prod
(for now you can just set defaults for example CLIENT_SECRET and ID set just empty string it is different, we need to get it from vault  -  we can do it later)
check how it is done in this pr https://github.com/RedHatInsights/insights-rbac/pull/1583/files